packages/app/src/components/Settings/Settings.js

```diff
@@ -232,7 +232,7 @@ const Settings = (props) => {
           className="settings__button"
           style={
             disableButton
-              ? { color: 'gray', 'border-color': 'gray', 'font-weight': 'bold' }
+              ? { color: 'gray', 'border-color': 'gray', 'font-weight': 'bold', 'opacity': '1' }
```

Comment:
These are the only changes in the file - any other changes from the previous commits are just prettier styling changes.